service/src/main/java/bio/terra/cbas/dao/MethodDao.java

```diff
@@ -28,29 +29,47 @@ public MethodDao(NamedParameterJdbcTemplate jdbcTemplate) {
     this.jdbcTemplate = jdbcTemplate;
   }
 
+  public static final String METHOD_JOIN_GITHUB_METHOD_DETAILS =
```

Comment:
Will this cause errors when trying to fetch Dockstore methods?

Reply:
Dockstore methods will not have a github method details so would look like any other pre-backfilling method. We just need to be aware that the values might be null and fill in an Optional.empty value for the details. There are some test cases for this too, which test this out for real in containerized tests.